easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'BioKanga'
+version = '4.3.4'
+
+homepage = 'https://github.com/csiro-crop-informatics/biokanga'
+description = """BioKanga is an integrated toolkit of high performance bioinformatics
+  subprocesses targeting the challenges of next generation sequencing analytics.
+  Kanga is an acronym standing for 'K-mer Adaptive Next Generation Aligner'."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['https://github.com/csiro-crop-informatics/biokanga/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['BioKanga-4.3.4_configure.patch']
+
+builddependencies = [
+    ('Autoconf', '2.69'),
+    ('Automake', '1.15')
```

Comment:
@nathanhaigh Do you mind changing this to `Autotools` `20150215`? You can either use the existing `Autotools-20150215-GCC-4.9.3-2.25.eb` easyconfig for that (which EasyBuild v3.x will pick up automatically when using `foss/2015b`), or add the additional necessary easyconfigs for `Autotools-20150215-foss-2015b.eb`